### ISSUE TITLE
RPN CONV 3*3 and cls_logits

### ISSUE THREAD

**Issue by 28150722** [facebookresearch/maskrcnn-benchmark#775]
## ❓ Questions and Help
I am trying to build a new Faster RCNN model. Since I am using the resnet101 C4 mode, I have to rebuild the model. But when I checked my model structure, I found some problems with the RPN head.
In the code, the input channel and output channel of the first 3X3 convolutional layer in the RPN are the same 1024 size. Should I change the output channel to 512?
Another question is whether the output channel of cls_logits should be num_anchors? Why is the output channel not num_anchors * 2 ?
When I try to train my own dataset, the loss is 'nan', I reduce the learning rate to 0.00001, and the batchsize is 2, then the training starts, but the rpn box reg loss can only converge to 1. This is normal. ?
I only have one GPU 1080Ti.
Thank you
